### PR TITLE
Try to use old mamba version

### PR DESCRIPTION
**Pull request by mathbunnyru**

### CODE REVIEW

File: base-notebook/Dockerfile [hunk 1]
```diff
@@ -81,7 +81,11 @@ WORKDIR /tmp
 # CONDA_MIRROR is a mirror prefix to speed up downloading
 # For example, people from mainland China could set it as
 # https://mirrors.tuna.tsinghua.edu.cn/github-release/conda-forge/miniforge/LatestRelease
-ARG CONDA_MIRROR=https://github.com/conda-forge/miniforge/releases/latest/download
+
+# Using fixed version of mamba, because the latest one has problems with arm under qemu
+# See: https://github.com/jupyter/docker-stacks/issues/1539
+ARG CONDA_MIRROR=https://github.com/conda-forge/miniforge/releases/download/4.10.3-7
```
Comment (minrk) [inline base-notebook/Dockerfile:87]: This sets the _initial_ mamba version, but the `conda update --all` in the next step always upgrades mamba to latest.

Comment (mathbunnyru) [reply base-notebook/Dockerfile:87]: You're right, I will solve this another way.

**Comment by mathbunnyru** [comment]
My approach should work, but we have a much bigger problem.

https://github.com/jupyter/docker-stacks/runs/4493047317?check_suite_focus=true#step:7:4475
```
#5 [1/4] FROM docker.io/jupyter/minimal-notebook@sha256:5d8ba694b92d9fe5802529b7ccf8bad23de6632d0bbaa92fc3fdc8a31cdc9c9c
4476
  #5 resolve docker.io/jupyter/minimal-notebook@sha256:5d8ba694b92d9fe5802529b7ccf8bad23de6632d0bbaa92fc3fdc8a31cdc9c9c done
4477
  #5 sha256:2f66eefcb707429dfa3d1b8d0cdcbf330bb3d2624db2821eb2eb2e7323e0358a 1.48kB / 1.48kB 0.0s done
4478
  #5 sha256:fd44b7e34afa5f14902e0239880773be42ca449f5d1dbab726b4a17cde4f48f4 41.11MB / 235.96MB 0.3s`
```

Because we build so many images, they take too much space and we end up pulling images from dockerhub.
Which means we're effectively building not the code which is at the current branch, but some mix of pas image and the new code.

**Comment by mathbunnyru** [comment]
@consideRatio I will for now completely disable arm images building at all :(
I think approach without multi-arch image will work better for us, it might be a little bit more confusing for the users, but I think it will work better for them as well.

**Comment by mathbunnyru** [comment]
Nice, the only `resolve docker.io` I see are `ubuntu:focal` images.
Let's get mamba pinning back, but this time I will only pin it for arm images.

**Comment by consideRatio** [comment]
> @consideRatio I will for now completely disable arm images building at all :(
I think the approach "build everything with some hacks in one place" doesn't work well for us.

That sounds reasonable. Given the limited maintenance capacity we have, and that this blocks everything if not maintained, cutting away something to save something else seems reasonable.

I think this strategy hasn't been robust enough given that each failure cause a lot of additional work and is hard to resolve due to the monolith CI jobs. I consider a resolution towards having arm again after disabling it would be to put in more work in making the CI system work in a more composable manner (#1407), I think that would make it handle multiple architectures in a more maintenance sustainable way.

---

Are you making some last attempts to get things to work atm?

**Comment by mathbunnyru** [comment]
Yes, let's wait for the latest build - it should run faster (because caching images works) and it gives expected result (because of caching).
It might be the issue with latest mamba, but it might be something else :(

**Comment by mathbunnyru** [comment]
Finally, this seems to work.
Let's merge this and hope it will work during push step.

**Comment by consideRatio** [comment]
@mathbunnyru it worked! You are amazing! Thank you for your thorough work!!! :heart: :tada:

**Pull request merged**